EncounterHandler.js

```diff
@@ -690,8 +690,12 @@ function inject_monster_image(stat) {
 	}
 	if (window.EncounterHandler.combat_body.find(".encounter-details-content-section__content .injected-image").length == 0) {
 		let content = window.EncounterHandler.combat_body.find(".encounter-details-content-section__content");
-		let image = `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
+		const image = stat.data.isHomebrew ?
+			`<img style="width:100%" class="injected-image" src="${stat.data.avatarUrl}"
+			alt="${stat.data.name}" class="monster-image" ;onerror=''></img>`
+		 	: `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
 			alt="${stat.data.name}" class="monster-image" onerror="this.src='${stat.data.avatarUrl}'";onerror=''></img>`;
```

Comment:
uhmm.  this also has two "onerror"

also.. on the receiving side everything is passed through DomPurify.. so any javscript is removed when sent through the gamelog

Reply:
Aye, so if the first fails it rolls over to the avatar images and if that fails it lands on the empty string to avoid infinite looping.

This doesn't pass the onerror down, but it does pass the correct src down when it's a homebrew rather than relying on the onerror.

Homebrew monsters 403 when you try to get the large image.

I do think the comment about the semicolons is correct though, I'll take a look in the morning

Reply:
Yeah, this part 

    onerror="this.src='${stat.data.avatarUrl}'";onerror=''

Should be 

    onerror="this.src='${stat.data.avatarUrl}';onerror='';"

The original onerror function needs to set the src and replace itself with an empty onerror function.

Reply:
Cheers, should now be sorted